paddle/fluid/framework/ir/mkldnn/cpu_quantize_pass.cc

```diff
@@ -877,6 +877,75 @@ void CPUQuantizePass::QuantizeElementwiseAdd(Graph* graph) const {
                   quantize_elementwise_add_count);
 }
 
+void CPUQuantizePass::QuantizeElementwiseMul(Graph* graph) const {
```

Comment:
Please check if you can parametrize this function because code for `QuantizeElementwiseAdd` and `QuantizeElementwiseMul` should be the same, so you can just add type for `QuantizeElementwise(Graph* graph, const std::string elementwise_type)`